### PR TITLE
Adds logo and a few other missing features for am-flughafen

### PR DESCRIPTION
**Pull request by cameronviner**
#### What does this PR do? (please provide any background)

Cleans up the am-flughafen brands by fixing the styling for:

- Logo - https://cloudup.com/cZuzevJj1Cx
- Modal - https://cloudup.com/cfZ9r5V4HVB
- Small text size - https://cloudup.com/c5cC1Qg8s1u

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by stuartgiles** [comment]
When reviewing your changes on http://localhost:8687/patterns/navbar-logged-in/index.html. I spotted the navbar menu toggle button is a bit too big which forces the navbar to be too big on mobile.

You should be able to override these variables - https://github.com/holidayextras/brands/blob/master/brands/globals/src/less/variables.less#L122
To adjust the element height.

**Comment by stuartgiles** [comment]
Looking at http://localhost:8687/elements/buttons/index.html

Looks like you might also need to change the button border colour.
`@btn-default-border: @btn-default-bg;`

File: brands/am-flughafen/src/less/navs.less [hunk 1]
```diff
@@ -1 +1,18 @@
-@import '../../../brand-globals/src/less/navs';
\ No newline at end of file
+@import '../../../brand-globals/src/less/navs';
+
+.nav-tabs {
```
Comment (stuartgiles) [inline brands/am-flughafen/src/less/navs.less]: What's the PR etiquette at HX? How pedantic should I be?
Your code will work fine, so this isn't a blocker, but I'd suggest matching the specificity of bootstrap (so your code does also affect any nested links) and also removing the duplication.

```
.nav-tabs {
  > li {
    > a {
      color: @color-white;
      background-color: @brand-primary;
      &:hover,
      &:focus {
        cursor: pointer;
        background-color: darken(@bg-primary, 10%);
      }
    }
  }
}
```

Comment (cameronviner) [inline brands/am-flughafen/src/less/navs.less]: Hey @stuartgiles I have now changed it to the above.

**Comment by stuartgiles** [comment]
Sorry, just remembered you should also bump the package version :-/

**Pull request merged**